Gifski/EditScreen.swift

```diff
@@ -101,6 +154,90 @@ struct EditScreen: View {
 		}
 	}
 
+	var aspectRatioPicker: some View {
```

Comment:
Would probably be better to move this to a separate view, so it can keep some state local. Or maybe even a separate view for the `ToolbarItemGroup` contents.